hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2286,7 +2291,8 @@
             to receive some entropy bits before the keys for SRAM ctrl and OTBN are computed.
             '''
       milestone: V2
-      tests: []
+      tests: ["chip_sw_sram_ctrl_ret_scrambled_access", "chip_sw_flash_init",
```

Comment:
@msfschaffner 
On some of these tests, the entropy source SS is not even initialized. Could we please confirm OTP actually fetches randomness from the ES before forwarding the keys to these memories (forgive my ignorance on this - i will read the spec too in the meantime)?

Reply:
This may be a problem.
OTP will always request entropy from EDN when a scrambling key is requested.
If the entropy complex is not properly initialized, this may hang forever.

I believe the SW team is aware of this and the ROM code should initialize ES beforehand, see:
https://github.com/lowRISC/opentitan/blob/8a46c174fed535900fa6a8cd8cd15b72e563636f/sw/device/silicon_creator/mask_rom/mask_rom_start.S#L206-L237

Something similar may have to be done in these tests here, too.

Reply:
actually from the wave, the OTP fetches EDN[1] and EDN indeed returns a valid (non-zero, non-all-ones, unique) value.

Reply:
Aah right, the test_rom replicates that piece of code from the mask rom. We are good here. In any case, at the chip level, we should add assertion checks to prove EDN always returns unique, non-zero, non-all1s data to all its requestors, just in case.